lib/engine/game/g_1848/game.rb

```diff
@@ -495,6 +520,7 @@ def setup
           @boe.cash = BOE_STARTING_CASH
           @stock_market.set_par(@boe, lookup_boe_price(BOE_STARTING_PRICE))
           @extra_tile_lay = false
+          @private_closed_triggered = false
```

Comment:
You don't need to add this because undefined is nil, which evaluates to false. It's fine to add...more of an FYI.